database/migrations/2021_02_12_083807_add_type_to_links.php

```
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class AddTypeToLinks extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('links', function (Blueprint $table) {
+            $table->string('type')->after('label')->nullable(false)->index();
```

to be honest - here I am just in doubts - whether not to set it to `->default('external')` ... 
but this is just a note. not a request. any reason?

---

A link without a `type` is not valid anymore. This way you have to be explicit about what type of link you're creating.

On DB side I prefer things to fail fast rather than silently accept incomplete input -- especially if the default is buried in a migration file.